### PR TITLE
#161747569 Get an article 

### PR DESCRIPTION
**Pull request by augustineezinwa**
#### What does this PR do?
- This PR fetches an article either by id or slug
#### Description of Task to be completed?
- add a controller to fetch an article
- add seed data
- add tests
#### How should this be manually tested?
- `npm test`
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- #161747569
#### Screenshots (if appropriate)
screen shot 2018-11-08 at 6 17 24 pm
screen shot 2018-11-08 at 6 18 10 pm
screen shot 2018-11-08 at 6 18 49 pm

#### Questions:
- None

### CODE REVIEW

File: server/controllers/ArticleController.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+              bio,
+              roleId
+            }
+          }
+        });
+      })
+      .catch(err => res.status(500).json({
+        status: 'failure',
+        errors: {
+          message: ['error reading Article\'s table', `${err}`]
```
Comment (mbilesanmi) [inline server/controllers/ArticleController.js]: Is this the error we'd be displaying to the user on the frontend?

Comment (augustineezinwa) [reply server/controllers/ArticleController.js]: Using  ``` .catch(err => res.status(500).json({
        status: 'failure',
        errors: err.message
      }));``` resolves to these 
     ```{
    "status": "failure",
    "errors": "column \"nan\" does not exist"
}```
while using these ```      .catch(err => res.status(500).json({
        status: 'failure',
        errors: {
          message: ['error reading Article\'s table', `${err}`]
        }
      }));``` resolves to these 
```{
    "status": "failure",
    "errors": {
        "message": [
            "error reading Article's table",
            "SequelizeDatabaseError: column \"nan\" does not exist"
        ]
    }
}```

None of the above errors makes sense to any user on the frontend. If we notice a status 500 error, we could choose to display `Internet connection error` or `server is down`. The above errors are just for development purposes and only makes sense to a developer.

**Comment by sulenchy** [comment]
LGTM.

File: package.json [hunk 2]
```diff
@@ -7,10 +7,10 @@
     "node": "10.11.0"
   },
   "scripts": {
-    "test": "npm run db-init && nyc mocha test/**/**/**.test.js --require @babel/register --require @babel/polyfill --exit",
-    "start": "npm run clean && npm run build && node dist/app.js",
+    "test": "npm run db-init && nyc mocha test/**/**/**.test.js --require babel-register --require babel-polyfill --exit",
+    "start": "npm run build && node --require babel-polyfill --require babel-register dist/app.js",
```
Comment (KvNGCzA) [inline package.json]: If our babel configurations are transpiling the files correctly then I do not think we still have to start our app with babel. This does not feel like a long term solution so I dont think we should merge this particular changes in. It's like transpiling and still starting the app with babel-node.

Comment (augustineezinwa) [reply package.json]: You are right. Let me fix that.

**Comment by KvNGCzA** [comment]
@mbilesanmi LGTM

**Pull request merged**